### ISSUE TITLE
Missing serum_multisig.so and serum_multisig.json file in v0.9.0 release

### ISSUE THREAD

**Issue by Rini-Tikki** [project-serum/multisig#48]
@armaniferrante I am working with the latest code of multisig, it would be great if someone can add these files in the latest release. TIA

**Comment by fanatid** [comment]
Have these files existed in previous releases? Why you can not build multisig files in your project?

**Comment by Rini-Tikki** [comment]
Yes, these files exist in all the previous releases. (like https://github.com/project-serum/multisig/releases/tag/v0.8.0). Just need these two files in my project to run a txn with multisig. For now, I have built it and put it directly in my project. But later, will need to write the script to download it from `releases`
